overrides/default.nix

```diff
@@ -888,6 +888,13 @@ lib.composeManyExtensions [
         }
       );
 
+      fastapi-cli = prev.fastapi-cli.overridePythonAttrs (
+        old: lib.optionalAttrs (!(old.src.isWheel or false)) {
```

Comment:
This probably needs to be done regardless of whether `fastapi-cli` is a wheel, right?

Reply:
Ah, yes, correct, I've copied that override block from another expression and didn't check this part again.

Reply:
Done